samples/Directory.Build.props

```diff
@@ -17,11 +17,13 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Content Include="..\..\src\Datadog.Trace.ClrProfiler.Native\bin\**"
+    <None Include="..\..\src\Datadog.Trace.ClrProfiler.Native\bin\$(Configuration)\\$(Platform)\\**"
```

Comment:
You can probably change the end here to `**\*.dll` so it only copies the dll files.

Reply:
what about `*.so`?

Reply:
The samples (and any profiled app) only need `Datadog.Trace.ClrProfiler.Native.dll`.

Reply:
Never mind me. I forgot about Linux.